test/e2e/smoke-test/packages/cert-manager/1.5.3/cert-manager-test.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+
+# Checking package is installed or not
+tanzu package installed list | grep "cert-manager.community.tanzu.vmware.com"
+
+exitcode=$?
+
+if [ "${exitcode}" == 1 ] 
+then
+version=$(tanzu package available list cert-manager.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+tanzu package install cert-manager --package-name cert-manager.community.tanzu.vmware.com --version "${version}"
+sleep 30s
+fi
+
+
+# Providing prerequisite 
+cat <<EOF | kubectl apply --filename -
```

Comment:
This test could also use a random test namespace like mentioned in https://github.com/vmware-tanzu/community-edition/pull/2246/files?authenticity_token=450WwLofW6a7J2Faj%2BDF9zLMPoeRGfikVXHqHGJQ2UNAYdPOmDh0aCxdnGOpjeP3aEufBmSe2Jsw8THsAqKXGA%3D%3D&file-filters%5B%5D=.sh&file-filters%5B%5D=.yaml#r741061685